src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      data-qa="big">
+      <div class="container--big">
+        <img
+          class="image image--big"
+          src="images/Search.svg" alt="search_button">
+        <label>
+          <input
+            class="form_input input--big"
+            type="text"
+            placeholder="Try &quot;Los Angeles&quot;"
+            data-qa="keypress"
+          >
+        </label>
+      </div>
+    </form>
+
+    <form
+      action="#"
+      method="post"
+      data-qa="small">
+      <div class="container--small">
+        <img
+          class="image image--small"
+          src="images/Search.svg" alt="search_button">
+        <label>
+          <input
+            class="form_input input--small"
+            type="text"
+            placeholder="Try &quot;Los Angeles&quot;"
+          >
```

Comment:
no need --big modifier in all classes, use default styles and then for --small only style the difference
checklist #2 https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md